### PR TITLE
Add conversion functions.

### PR DESCRIPTION
**Pull request by bitbckt**

### CODE REVIEW

File: lib/fauna/query.rb [hunk 1]
```diff
@@ -820,6 +820,30 @@ def not_(boolean)
       Expr.new not: Expr.wrap(boolean)
     end
 
+    ##
+    # Converts an expression to a string literal.
```
Comment (erickpintor) [inline lib/fauna/query.rb]: We should add the link to their section in the docs, if there is a link.

File: spec/query_spec.rb [hunk 2]
```diff
@@ -1039,6 +1039,38 @@ def to_hash
     end
   end
 
+  describe '#to_string' do
+    it 'serializes' do
```
Comment (erickpintor) [inline spec/query_spec.rb]: I think this suite is meant for functional tests.

File: lib/fauna/query.rb (hunk 1 above)
Comment (bitbckt) [reply lib/fauna/query.rb]: There isn't one yet.

File: spec/query_spec.rb (hunk 2 above)
Comment (bitbckt) [reply spec/query_spec.rb]: I noticed that; it's odd that some drivers seem to only have functional tests, where others have serialization.

I take it you prefer I rewrite as a functional test?

Comment (marrony) [reply spec/query_spec.rb]: If I not mistake all drivers has functional tests, but not all has serializations

Comment (bitbckt) [reply spec/query_spec.rb]: Serialization makes more sense to me, in that the semantics of the language is defined and tested in the database. The clients need only conform to the wire protocol.

That said, I'll rewrite these as functional tests.

Comment (marrony) [reply spec/query_spec.rb]: I'm afraid only conform to wire protocol isn't not enough, we catch a lot of edge cases while developing the drivers because of the functional tests, those tests force us to really utilize the api and see if the functions fit together. One exemple was recursive references, I found out there wasn't a way to paginate over nested objects when I had to "paginate" those objects.
